.github/workflows/default_on_push.yml

```
@@ -11,6 +11,7 @@ on:
       - .gitmodules
       - Dockerfile
       - setup.sh
+      - VERSION # also update :edge when a release happens
```

The comment is unnecessary:

```suggestion
      - VERSION
```

`VERSION` contributes no meaning to `:edge`. In the past we have once [missed bumping `VERSION` before tagging a release](https://github.com/docker-mailserver/docker-mailserver/issues/2952#issuecomment-1364304849), but since we [only publish images other than `:edge` by tagged commits](https://github.com/docker-mailserver/docker-mailserver/pull/2944#issuecomment-1364361134), we just published a new tag semver-patch release to resolve that.

I'd rather see the [`VERSION` file removed since it's unnecessary to support the functionality it's intended for](https://github.com/docker-mailserver/docker-mailserver/pull/3641#issuecomment-1826483637), but since it the [script in all prior releases since v10](https://github.com/docker-mailserver/docker-mailserver/pull/1951) will rely on checking `VERSION` we'd need to keep it around for a while 🤷‍♂️ 

---

If the better update check approach is taken, then the addition to build an image when `VERSION` changes would become redundant:

```suggestion
```